src/components/welcome.jsx

```diff
@@ -3,9 +3,13 @@ import {Link} from 'react-router'
 
 export default _ => (
   <div className='box-body'>
-    <h3>Welcome to FreeFeed</h3>
-    <p>&lt;Here's the description of what FreeFeed is. We need a copywriter!&gt;</p>
-    <p>If you already have an account, you might want to <Link to='/signin'>sign in</Link>.</p>
-    <p>Otherwise, consider <Link to='/signup'>signing up</Link> to enjoy all the fun stuff.</p>
-  </div>
+    <h3>Welcome to FreeFeed.net</h3>
+      <p>FreeFeed is an <a href="https://github.com/FreeFeed/" target="_blank">open-source</a> social
```

Comment:
We don't need trailing slash in the URL. And I'd argue we need that link here at all. The two important links in this text must be "Sign up" and "Sign in" (and maybe "About", but only barely), so we don't want to make them more competition with a GitHub link.